app/uk/gov/hmrc/incometaxsessiondata/models/MtdItUser.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsessiondata.models
+
+import play.api.libs.json.{Format, Json}
+import uk.gov.hmrc.auth.core.AffinityGroup
+
+case class MtdItUser(mtditid:   String,
```

Comment:
session instead of MtdItUser
+ if we can rename models folder into domain, as this would be representing what this service can do/return in scope of this domain;
internal wiring/models would be in the models as usual